### PR TITLE
fix: usr: #253: Reduce connect and read time out for client and server

### PR DESCRIPTION
**Pull request by abhishekdas99**

### CODE REVIEW

**Comment by kvankayala** [comment]
@jordanw-bq @abhishekdas99  Codecov report shows that the code coverage will decrease slightly with this PR.
But I don't see any major change that effects the code coverage number.
Do we have to check codecov configuration again?

**Comment by jordanw-bq** [comment]
@kvankayala I think we'll need to take a look at the Cobertura logs around `ClusterType` and `Hadoop2ClusterManager`, since these often seem to have unexpected coverage changes.

File: rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java [hunk 1]
```diff
@@ -228,7 +228,7 @@ public static int getFileValidationInterval(Configuration conf)
 
   public static int getClientTimeout(Configuration conf)
```
Comment (jordanw-bq) [inline rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java]: Change the CacheConfig method names to reflect the config option changes.

Comment (abhishekdas99) [reply rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java]: Done

**Comment by jordanw-bq** [comment]
LGTM 👍

File: rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java [hunk 2]
```diff
@@ -311,9 +311,9 @@ public static int getServiceRetryInterval(Configuration conf)
     return conf.getInt(KEY_SERVICE_RETRY_INTERVAL, DEFAULT_SERVICE_RETRY_INTERVAL);
   }
 
-  public static int getSocketReadTimeOut(Configuration conf)
+  public static int getClientReadTimeOut(Configuration conf)
```
Comment (jordanw-bq) [inline rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java]: Method name should be `...Timeout(...` for consistency

**Pull request merged**